package.json

```diff
@@ -1,55 +1,23 @@
 {
-  "name": "@vercel/fetch",
-  "version": "6.1.1",
-  "description": "Opinionated `fetch` optimized for use inside microservices",
-  "license": "MIT",
-  "main": "index.js",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "lint": "eslint .",
-    "lint:staged": "lint-staged",
-    "test": "yarn run lint && best --verbose"
+    "build": "turbo run build",
+    "dev": "turbo run dev --parallel",
+    "lint": "turbo run lint",
+    "test": "turbo run test",
+    "clean": "turbo run clean && rm -rf node_modules",
+    "format": "prettier --write \"**/*.{js,md}\""
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/vercel/fetch.git"
-  },
-  "author": "Nathan Rajlich <nate@vercel.com>",
-  "bugs": {
-    "url": "https://github.com/vercel/fetch/issues"
-  },
-  "homepage": "https://github.com/vercel/fetch#readme",
-  "dependencies": {
-    "@types/async-retry": "1.2.1",
-    "@vercel/fetch-cached-dns": "^2.0.2",
-    "@vercel/fetch-retry": "^5.0.2",
-    "agentkeepalive": "3.4.1",
-    "debug": "3.1.0"
-  },
-  "peerDependencies": {
-    "@types/node-fetch": "2",
-    "node-fetch": "2"
-  },
-  "pre-commit": "lint:staged",
-  "lint-staged": {
-    "*.js": [
-      "eslint",
-      "git add"
-    ]
+  "devDependencies": {
+    "prettier": "^2.5.1",
+    "turbo": "latest"
   },
-  "eslintConfig": {
-    "extends": "@zeit/eslint-config-node",
-    "rules": {
-      "linebreak-style": 0
-    }
+  "engines": {
+    "npm": ">=7.0.0",
+    "node": ">=14.0.0"
```

Comment:
This doesn't match the tests wich are running node 12

Reply:
Ahh good catch - this was auto generated or copied from the turborepo I based this off of.